vignettes/test_data_guidance.Rmd

```diff
@@ -0,0 +1,22 @@
+---
+title: "Test Data Guidance"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Test Data Guidance}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+There are two test data packages that enable any Pharmaverse package development team to re-use test SDTM and ADaM: 
+
+- [`{pharmaversesdtm}`](https://github.com/pharmaverse/pharmaversesdtm): a data package that stores test SDTM data. If the current available data does not meet your need, update or add the new data in this package based on the instructions provided on the [how to update page](https://pharmaverse.github.io/pharmaversesdtm/cran-release/) 
```

Comment:
can the link text be updated to "How to Update" please. Also, are you waiting on the page to go live before you add the link to it? I think that makes sense but perhaps you could add the link to it anyway, knowing it won't work for now?